### DISCUSSION TITLE
Custom workflows like the new auto-archive workflow

### DISCUSSION THREAD

**Discussion by aarongable** [community/community#53705] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

The new auto-archive and auto-add workflows are wonderful. I was previously using a GitHub Action to do auto-add, and it's really nice to not have to maintain a PAT for that Action to have access to the project anymore.

So now I'd like to request the logical next step: custom workflows that behave like the auto-archive workflow. Specifically, I want to be able to:
- Specify a custom search filter, like "is:open updated:<@today-2w", or (using custom fields from the project) "-status:untriaged last-triaged:<@today-12w"
- Specify multiple custom actions to take, such as "Set Value: status:untriaged" + "Set Value: last-triaged:none"

It seems like all of the pieces for this -- workflows that set custom project values, workflows that run periodically and search for items to process -- are in place. All we need now is a little bit more customizability and we'll be off to the races!

Thanks so much for all the work so far; I'm really looking forward to the next set of Projects improvements.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jeherve** [comment]
Related discussions:

- https://github.com/orgs/community/discussions/55096
- https://github.com/orgs/community/discussions/45653

**Comment by probitcarwyn** [comment]
This is on the public roadmap here:

- https://github.com/github/roadmap/issues/281

**Comment by aarongable** [comment]
The public roadmap issue for this feature has been closed as "outdated", and no replacement issue has been added to the roadmap. Can we get any confirmation that this feature is even being considered? It is, after all, still prominently advertised on [the GitHub Issues and Projects homepage](https://github.com/features/issues):
<img width="1285" height="773" alt="image" src="https://github.com/user-attachments/assets/70c39cbe-2d36-49c5-89b9-a2cde4366863" />

**Comment by kayson827363636636363-hub** [comment]
That’s a solid suggestion  having more customizable workflows would streamline processes even further, much like how a[ b2b lead generation usa agency](https://neelummuneera-droid.github.io/b2b-lead-generation-agency-usa/) optimizes automation to refine targeting and efficiency.
